api/filter/network/http_connection_manager.proto

```diff
@@ -80,6 +80,23 @@ message HttpConnectionManager {
     // populate the tag name, and the header value is used to populate the tag value. The tag is
     // created if the specified header name is present in the request's headers.
     repeated string request_headers_for_tags = 2;
+
+    // Target percentage of requests managed by this HTTP connectionmanager that will be force
+    // traced if the *x-client-trace-id* header is set. Defaults to 100%. This variable is a direct
+    // analog for the variable of the same name in the :ref:`HTTP Connection Manager
+    // <config_http_conn_man_runtime>`.
+    double client_sampling = 3 [(validate.rules).double = {gte: 0, lte: 100}];
+
+    // Target percentage of requests managed by this HTTP connection manager that will be traced
+    // after all other checks have been applied (force tracing, sampling, etc.). Defaults to 100%.
+    // This variable is a direct analog for the variable of the same name in the :ref:`HTTP
+    // Connection Manager <config_http_conn_man_runtime>`.
+    double sampling = 4 [(validate.rules).double = {gte: 0, lte: 100}];
```

Comment:
Can you start a `STYLE.md` in root with a single bullet point for now, which captures the point that @wora made about preferring `double` over `float` for JSON conversion reasons? This is is something we should now start to be more consistent about, and it's one of these missing proto3 style guidelines that would be good to codify. Thanks!